signalfx-forwarder-app/app.manifest

```diff
@@ -5,7 +5,7 @@
     "id": {
       "group": null, 
       "name": "signalfx-forwarder-app", 
-      "version": "2.0.1"
+      "version": "2.0.2"
```

Comment:
Not totally sure of the versioning practices here, but a new feature would warrant `2.1.0` imo.

Reply:
Agreed - will push with a new version